src/test/regress/specs/isolation_ensure_dependency_activate_node.spec

```diff
@@ -20,6 +20,21 @@ teardown
     DROP TABLE IF EXISTS t2 CASCADE;
     DROP TABLE IF EXISTS t3 CASCADE;
     DROP TYPE IF EXISTS tt1 CASCADE;
+    DROP FUNCTION IF EXISTS add(INT,INT) CASCADE;
+
+    -- Remove the pg_dist_object record manually as we do not yet hook into DROP FUNC
```

Comment:
did you verify this was required?

the citus drop trigger should already remove the entry from `pg_dist_object`

Reply:
> did you verify this was required?

Yes, I needed to do this as the next permutations increased the total number of entries in the `pg_dist_object` table after `create_distributed_function()` calls for the same function in the test.

> the citus drop trigger should already remove the entry from pg_dist_object

I do not think it did.

> Also, is there any moment the casting doesn't fail? The function just has been deleted one line above

Tough question, will check it out and report it here